packages/osd-stylelint-plugin-stylelint/src/utils/is_color_property.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+const COLOR_PROPERTIES = [
```

Comment:
Can we add a comment here that explains why/how these properties were chosen, and how it's intended to be used?